inc/TRestTrackLineAnalysisProcess.h

```diff
@@ -36,6 +36,8 @@ class TRestTrackLineAnalysisProcess : public TRestEventProcess {
     /// A pointer to the output event Track event
     TRestTrackEvent* fOutTrackEvent;
 
+    std::string fLineAnaMethod = "default";
```

Comment:
The default is that it will be streamed, but I would add at the end `//<`. On the same sense, I believe it is better to mark the pointers above `TRestTrackEvent *fTrackEvent; //!` to avoid writing the pointer to disk.

Reply:
Done.
Since you bring up this issue, what about the functions inside the header, shall we add `//!`, I think I have seen some weird stuff inside the root files.